lib/wab/impl/init.rb

```diff
@@ -30,7 +30,9 @@ def setup(path, config)
           write_opo_rub_conf(config_dir, types)
 
         rescue StandardError => e
-          puts "*-*-* #{e.class}: #{e.message}"
+          # TBD: Issue more helpful error message
+          puts %|*-*-* #{e.class}: #{e.message}\n      #{e.backtrace.join("\n      ")}|
+          abort
```

Comment:
This isn't really necessary as the application is done anyway.

Reply:
This is necessary because when a StandardError is raised, it gets rescued, prints the event message and then *continues* on to print the success message from `wabur` which is incorrect in this case..